src/components/cylc/GraphSubgraph.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <g class="c-graph-subgraph">
+    <rect
+      :width="subgraph.width"
+      :height="subgraph.height"
+      :x="subgraph.x"
+      :y="subgraph.y"
+      :visibility="subgraph.label=='margin' ?  'hidden' : 'visible' "
```

Comment:
In order to force a margin between the subgraphs Ive put an extra subgraph in with visibility set to hidden

Used the solution mentioned here:
https://stackoverflow.com/questions/9871062/padding-between-cluster-boundaries-and-nodes-when-using-graphviz-and-neato

Reply:
Could we avoid creating a `<rect />` element for the hidden subgraph?

Not a big deal, but worth cutting down on the number of elements where possible.